riocli/deployment/model.py

```diff
@@ -54,11 +53,16 @@ def find_object(self, client: Client) -> typing.Any:
         return obj if guid else False
 
     def create_object(self, client: Client, **kwargs) -> typing.Any:
-        pkg_guid, pkg = self.rc.find_depends(self.metadata.depends,
-                                             self.metadata.depends.version)
+        pkg_guid, pkg = self.rc.find_depends(
+            self.metadata.depends,
+            self.metadata.depends.version)
 
         if pkg_guid:
             pkg = client.get_package(pkg_guid)
+
+        if not pkg:
+            raise ValueError('package not found: {}'.format(self.metadata.depends))
```

Comment:
suggestion: add version also, as there can be package with same name but missing version.

Reply:
`self.metadata.depends` will print both the package name and version, right?